frontend/app/routes/public/renew/$id/ita/confirm-address.tsx

```diff
@@ -54,11 +56,17 @@ export async function action({ context: { session }, params, request }: ActionFu
   const t = await getFixedT(request, handle.i18nNamespaces);
 
   const hasAddressChangedSchema = z.object({
-    hasAddressChanged: z.nativeEnum(HasAddressChangedOption, {
+    hasAddressChanged: z.nativeEnum(AddressRadioOptions, {
       errorMap: () => ({ message: t('renew-ita:confirm-address.error-message.has-address-changed-required') }),
     }),
   });
 
+  const areMailingAndHomeAddressSameSchema = z.object({
+    areMailingAndHomeAddressSame: z.nativeEnum(AddressRadioOptions, {
+      errorMap: () => ({ message: t('renew-ita:confirm-address.error-message.are-mailing-and-home-address-same-required') }),
+    }),
+  });
+
```

Comment:
Thoughts on combining this Zod schema with the `hasAddressChangedSchema` above? That way we only need to parse and handle the result of a single schema.

Reply:
probably.  Imma wait until the designers respond to my comment in Figma.